packages/components/src/local/organisms/planning-channel/helpers/OrderEditing.tsx

```diff
@@ -112,7 +112,10 @@ export const OrderEditing: React.FC<OrderEditingProps> = ({ saved, activityBeing
 
       layerToEdit.on('pm:edit', e => {
         const pointsOnLine = get(e, 'layer._rings[0]') as Point[]
-        const preventRemove = !!(e.shape.toLowerCase() === 'line' && pointsOnLine.length <= 2)
+        const preventRemove = !!(
```

Comment:
Hopefully I am wrong, but I _don't think_ this will fix it.  I _think_ this logic is only called when editing starts.  In the above reproducible steps, there are enough points at the start of editing, it's during editing that problem occurs.